scripts/apply_patches.sh

```diff
@@ -6,6 +6,33 @@ CDIR="$(cd "$(dirname "$0")" ; pwd -P)"
 XDIR=$CDIR/..
 PTDIR=$XDIR/..
 
+TORCH_PIN="$XDIR/torch_patches/.torch_pin"
+if [ -f "$TORCH_PIN" ]; then
+  CID=$(cat "$TORCH_PIN")
+  # If starts with # and it's not merged into master, fetch from origin
+  if [[ $CID = \#* ]]; then
+    PRNUM="${CID//[!0-9]/}"
+    set +x
```

Comment:
oh yeah we have `set -ex` set globally in the beginning already. 
But since output of `$(git -C $PTDIR log -1000)` is so long that I actually disable printing here.